africanlii/views/home.py

```diff
@@ -19,7 +19,12 @@ def get(self, request, *args, **kwargs):
         recent_documents = GenericDocument.objects.order_by("-date")[:5]
         recent_instruments = LegalInstrument.objects.order_by("-date")[:5]
         recent_legislation = Legislation.objects.order_by("-date")[:5]
-        documents_count = GenericDocument.objects.count()
+        documents_count = (
+            Judgment.objects.count()
+            + GenericDocument.objects.count()
+            + LegalInstrument.objects.count()
+            + Legislation.objects.count()
+        )
```

Comment:
@musangowope This will count all the documents okay, or you could use CoreDocumentModel.objects.count(), (one query). Also I have noticed that the recent documents above are ordering by "date" but we want "updated_at"